interfaces/Portalicious/eslint.config.js

```diff
@@ -76,6 +76,13 @@ module.exports = tseslint.config(
       eslintPluginPrettierRecommended,
     ],
     rules: {
+      '@angular-eslint/template/i18n': [
+        'error',
+        {
+          checkId: false,
```

Comment:
Optionally we could enforce the usage of ids, rather than using auto-generated ones. This would also play nicely with the defaults of [daniel-sc/ng-extract-i18n-merge](https://github.com/daniel-sc/ng-extract-i18n-merge). However, it also makes our source files a bit more verbose.

I am in favour of doing less so I'd prefer auto-generated ids, but I don't have a strong preference and I'm easily persuaded.

Reply:
I like that using "ID you have to name yourself" allows to use "names that make sense"... but the downside is "you _have_ to name them yourself". :D

I wonder if these auto-generated IDs will give us noisy/ugly diffs in day-to-day work... Then I think we have a good argument to "do it better by hand", maybe.

Reply:
A "realistic" diff would be something like:
https://github.com/global-121/121-platform/pull/5568/commits/33f5c6c1b023e07c5f06b1db63b2067d72d6e7d3#diff-1e9eba834e433eeec035982534cf84aa6206b45805b53ff97ddff026a82751a1

So adding/removing some bits... It doesn't look _that_ bad.. ;)